rskj-core/src/main/java/co/rsk/pcc/NativeMethod.java

```diff
@@ -77,7 +77,7 @@ public long getGas(Object[] parsedArguments, byte[] originalData) {
         // the default can always be used on top.
         // (e.g., "return 23000L + super(parsedArguments, originalData);")
 
-        return originalData == null ? 0 : originalData.length * 2;
+        return originalData == null ? 0 : originalData.length * 2L;
```

Comment:
don't we also need `0L` here?

Reply:
The thing is that "0" can be automatically cast to long without causing overflows or data loss. The problem in this case was with the multiplication only, because by being the two factors of int type, the result could cause an overflow, so, converting one operand to long, eliminates the possibility of such overflow.